src/tables/cmap.js

```diff
@@ -178,12 +201,17 @@ function parseCmapTable(data, start) {
     const p = new parse.Parser(data, start + offset);
     cmap.format = p.parseUShort();
 
-    if (cmap.format === 12) {
+    if (cmap.format === 0) {
+        parseCmapTableFormat0(cmap, p, platformId, encodingId);
+    } else if (cmap.format === 12) {
         parseCmapTableFormat12(cmap, p);
     } else if (cmap.format === 4) {
         parseCmapTableFormat4(cmap, p, data, start, offset);
     } else {
-        throw new Error('Only format 4, 12 and 14 cmap tables are supported (found format ' + cmap.format + ').');
+        throw new Error(
```

Comment:
Template strings may be more appropriate/readable ?

Reply:
I agree, but right now we use string concatenation everywhere. Does esbuild take care of string literals for backward compatibility?

Reply:
I don't this esbuild is transpiling any code.
I this case we can merge as it is :+1: